### PR TITLE
Fix #75: port separation

### LINKED ISSUE #75: Fix route to use graphiql
`http://localhost:5555/graphiql` is routed to the UI instead of graphiql

### PR DESCRIPTION
**Pull request by frankobe**
Fix #75 

1. change url to `127.0.0.1`

2.separate web host port & api port 

3.add maxAge=0

### CODE REVIEW

**Comment by dwalintukan** [comment]
can you rebase this with master? I removed all the incognito logic for `openBrowser()`

**Comment by frankobe** [comment]
@dwalintukan wonder is there still necessary

File: src/index.js [hunk 1]
```diff
@@ -84,42 +89,43 @@ function startQtumProcess(reindex) {
...
     directory: path.join(__dirname, '../ui'),
     default: 'index.html',
+    maxAge: 0,
   }));
 
+  syncRouter.applyRoutes(apiServer);
+  apiRouter.applyRoutes(apiServer);
+
+  apiServer.listen(5555, () => {
+    logger.debug('Bodhi Api is running on http://127.0.0.1:5555.');
```
Comment (dwalintukan) [inline src/index.js]: API still running on 5555? not on 5000?

File: src/route/sync.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ syncRouter.post('/graphql', graphqlRestify(graphQLOptions));
 
 syncRouter.get('/graphiql', graphiqlRestify({
   endpointURL: '/graphql',
-  subscriptionsEndpoint: `ws://localhost:${syncPort}/subscriptions`,
+  subscriptionsEndpoint: `ws://127.0.0.1:${syncPort}/subscriptions`,
```
Comment (dwalintukan) [inline src/route/sync.js]: set const of domain: 127.0.0.1 in `config.js`?

File: src/index.js (hunk 1 above)
Comment (frankobe) [reply src/index.js]: @dwalintukan it is correct, app on 5000, api on 5555

**Pull request closed without merging**